lib/aws/rails/railtie.rb

```diff
@@ -64,5 +68,21 @@ def self.instrument_sdk_operations
         end
       end
     end
+
+    # Register a middleware that will handle requests from the Elastic Beanstalk worker SQS Daemon.
+    # This will only be added in the presence of the AWS_PROCESS_BEANSTALK_WORKER_REQUESTS environment variable.
+    # The expectation is this variable should only be set on EB worker environments.
+    def self.add_sqsd_listener_middleware(app)
+      is_eb_worker_hosted = ENV['AWS_PROCESS_BEANSTALK_WORKER_REQUESTS']
```

Comment:
You can use the `Aws::Util.str_2_bool` method for this.

Reply:
Sounds good - will do.